FLT/NumberField/InfiniteAdeleRing.lean

```diff
@@ -137,4 +137,11 @@ def baseChangeEquiv :
   IsModuleTopology.continuousAlgEquivOfIsScalarTower K _ (baseChangeEquivAux K L)
     (by simp_rw [baseChangeEquivAux_apply]; exact SemialgHom.baseChange_of_algebraMap_tmul_right _)
 
+instance : IsScalarTower K (InfiniteAdeleRing K) (InfiniteAdeleRing L) :=
```

Comment:
Again, do you think that we should even have InfiniteAdeleRing L being a K-algebra?